src/kaltura-player.js

```diff
@@ -28,6 +28,28 @@ import {
   Utils
 } from '@playkit-js/playkit-js';
 
+type playersType = {[id: string]: Object};
```

Comment:
I don't see how this type is necessary...
But if so, it should be:
`type PlayerInstances = {[id: string]: KalturaPlayer};`

Reply:
If you still use this type, move it to `/flow-typed` and doc it with `@typdef`